README.md

```diff
@@ -1 +1,69 @@
-# microverse-javascript
\ No newline at end of file
+# Portfolio portfolio
+
+The project is inspired by the figma microverse template 1 (https://figma.com/)
+For this project about portfolio website , I implemented the mobile menu by using my recently acquired JavaScript knowledge.
+
+- I used basic JavaScript syntax.
+- I used JavaScript to manipulate DOM elements.
+- I used JavaScript events.
+
+![screenshot](./image/demo.PNG)
+
+ 
+
+## Live Demo
+
+[Live Demo Link](https://codecaiine.github.io/microverse-javascript/#contact)
+ 
+## Built With
+
+- HTML
+- CSS
+- Javascript
+- Bootstrap 
+- HTML&CSS Linter
+
+## Get Started
+
+Please get your browser update.
+To get a local copy up and running follow these simple steps.
+
+Run the command below from the Terminal:
+
+      git clone <URL> in your terminal
+
+	  cd microverse_portfolio
+
+	  start index.html
+
+
+
+## Authors
+
+👤 **Yannick-Noel AKA**
+
+- Location: Côte d'Ivoire
+- GitHub: [@codecaiine](https://github.com/codecaiine)
+- Twitter: [@yannicknaka](https://twitter.com/yannicknaka)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/yannick-no%C3%ABl-aka/)
+
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
```

Comment:
- [ ] Please add a proper issue page link. You should always try to make your project Readme perfect.